### ISSUE TITLE
sil loss problem

### ISSUE THREAD

**Issue by MMMMichaelzhang** [yl4579/PitchExtractor#4]
![Screenshot from 2022-06-19 22-30-27](https://user-images.githubusercontent.com/91544231/177025576-fd7d37ab-22ca-4f50-9e9e-9f2d1594ae82.png)
![Screenshot from 2022-07-03 13-01-58](https://user-images.githubusercontent.com/91544231/177025578-7148139b-5685-4a90-b95d-e63b2de59fa6.png)
I trained 3 times with different data, the data includes talking and singing, but the eval sil loss goes up each time. Can you help me with this problem? thank you very much. @yl4579

**Comment by skol101** [comment]
Have you tried using data_augmentation flag?

**Comment by yl4579** [comment]
Sorry for the late reply. I was pretty busy recently. This is likely due to some miscongfiguration [here](https://github.com/yl4579/PitchExtractor/blob/main/Configs/config.yml#L16). The proportion between F0 and silence loss is not well balanced so the silence loss starts to overfit. You may try to set `lambda_f0 = 0.01` or even smaller so they are on the same scale, or use a very big dataset like LibriTTS (the dataset that the pre-trained model was trained on).
